images/update-drivers-website/entrypoint.sh

```diff
@@ -31,14 +31,14 @@ function check_program {
 # Meant to be run in the https://github.com/falcosecurity/test-infra repository.
 main() {
     # Checks
-    check_program "awscli"
+    check_program "aws"
     check_program "update-drivers-website"
     
     # Build updated json
-    update-drivers-website /tmp/website
+    update-drivers-website images/update-drivers-website/
     
     # Push the updated files to s3 bucket
-    aws s3 cp "/tmp/website/" s3://${S3_DRIVERS_BUCKET}/${S3_DRIVERS_KEY_PREFIX}/site --recursive --include "*.json" --include "index.html" --acl public-read
+    aws s3 cp "images/update-drivers-website/" s3://${S3_DRIVERS_BUCKET}/${S3_DRIVERS_KEY_PREFIX}/site --recursive --include "*.json" --include "index.html" --acl public-read
```

Comment:
Instead, why not shipping the index.html artifact with the image, copying it from that and avoid decorating the prowjob with the repostiory? (if the index.html artifact is supposed to be versioned).

In any case, if it makes sense to you @FedeDP, is only an improvement.